grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -44,15 +44,13 @@ class OrderService {
     def getOrders(Order orderTemplate, Date dateOrderedFrom, Date dateOrderedTo, Map params) {
         def orders = Order.createCriteria().list(params) {
             and {
-                if (orderTemplate.name || orderTemplate.description) {
```

Comment:
@DominikaHatala What if `q` will be empty, but `orderTemplate` will have either `name` or `description`. Seems like it is still possible to pass name and description with the `orderTemplate` here: https://github.com/openboxes/openboxes/blob/develop/grails-app/controllers/org/pih/warehouse/order/OrderController.groovy#L68.
cc @jmiranda

Reply:
This is the only place where gerOrders() is used and after the change I've applied in this ticket it is no possible to pass name and description with the orderTemplate.